pages/views/examples/scrollIntoView.njk

```diff
@@ -0,0 +1,35 @@
+---
+layout: content
+title: ScrollIntoView
+seoTitle: Custom ScrollIntoView implementation
+name: ScrollIntoView
+tags: examples
+icon: examples/scrollbar.svg
+aside:
+  components:
+    - esl-scrollbar
+---
+
+<esl-scroll-controls for=".scrollable-test-lvl-1, .scrollable-test-lvl-2, .scrollable-test-lvl-3"></esl-scroll-controls>
+
+<div class="scrollable-test-lvl-3-wrapper border bg-white relative-block">
+  <div scrollname="scrollableWrapper" style="height: 1200px" class="scrollable-test-lvl-3 p-4 esl-scrollable-content">
+
+    <div class="scrollable-test-lvl-2-wrapper border bg-white relative-block">
+      <div scrollname="scrollableList" class="scrollable-test-lvl-2 p-4 esl-scrollable-content">
+        <ul class="scrollable-test-lvl-1-wrapper">
+          {% for i in range(1, 17) -%}
+            <li scrollname="{{ i }}" style="background: {{ functions.randomColor() }}" class="scrollable-test-lvl-1">
+              <p class="scrollable-test-p">{{ i }}</p>
+            </li>
+          {%- endfor %}
+        </ul>
+      </div>
+      <esl-scrollbar class="scrollbar-blue" target="::prev(.esl-scrollable-content)"></esl-scrollbar>
+      <esl-scrollbar class="scrollbar-blue scrollbar-second" target="::prev(.esl-scrollable-content)" horizontal></esl-scrollbar>
```

Comment:
The scroll is only one of the cases
The original issue resolution can not be closed until all the primary cases will be covered.